packages/cfd/src/_common/base/server_time.js

```diff
@@ -1,8 +1,8 @@
-import { PromiseClass } from '../utility';
+import { PromiseUtils } from '@deriv-com/utils';
 
 const ServerTime = (() => {
     let clock_started = false;
-    const pending = new PromiseClass();
+    const pending = PromiseUtils.createPromise();
```

Comment:
out of curiosity, why do we even need a class for a promise? sounds a bit excessive

Reply:
Who knows why they did that 😅

Reply:
@wojciech-deriv  it is not a `class` it is a module namespace. With regards to why: https://github.com/deriv-com/deriv-utils/blob/master/src/utils/promise.utils.ts have a look at the docs 👍🏼